powerscale/provider/synciq_peer_certificate_resource_test.go

```diff
@@ -83,10 +83,19 @@ func TestAccSyncIQPeerCertificateResource(t *testing.T) {
 					path = "/ifs/invalid.pem"
 				}
 				`,
+				PreConfig: func() {
+					if FunctionMocker != nil {
+						FunctionMocker.Release()
+					}
+					FunctionMocker = mockey.Mock(helper.CreatePeerCert).Return("", fmt.Errorf("mock errorr")).Build()
+				},
 				ExpectError: regexp.MustCompile(`.*Failed to create SyncIQ Peer Certificate.*`),
 			},
 			// create valid
 			{
+				PreConfig: func() {
+					FunctionMocker.Release()
```

Comment:
Should you check for nil here?

Reply:
Probably not required since the previous step sets the variable. We'll never get to this stage if previous step failed.